Makefile

```diff
@@ -270,14 +270,14 @@ test-e2e-cache:
 	$(MAKE) clean-e2e
 	$(MAKE) test-e2e-cache-btc-staking-pre-approval
 	$(MAKE) test-e2e-cache-ibc-transfer
-	$(MAKE) test-e2e-cache-bcd-consumer-integration
+	$(MAKE) test-e2e-bcd-consumer-integration
 #	$(MAKE) test-e2e-cache-upgrade-v1
 
 clean-e2e:
 	docker container rm -f $(shell docker container ls -a -q) || true
 	docker network prune -f || true
 
-test-e2e-cache-bcd-consumer-integration: start-bcd-consumer-integration
+test-e2e-bcd-consumer-integration: start-bcd-consumer-integration
```

Comment:
So, the issue or trade-off here is that this is not cached anymore?

We can still have a no-deps low level 'cache' target here if we want. The 'no-cache' target then calls the 'cache' option, which is there for convenience / user experience.

Reply:
Yes the cache was mostly for accelerating GH action, specifically `ci.yml` pipeline.

However, now I find this pipeline conflates a number of things, including unit tests, e2e for BTC staking, e2e for BTC timestamping, e2e for integration, etc.. Probably we will need to split it into multiple ones. After doing that we might not need this cache, because the docker images will only be used for BTC staking integration e2e. Wdyt?